src/transformers/models/auto/modeling_auto.py

```diff
@@ -21,8 +21,6 @@
 from ...configuration_utils import PretrainedConfig
 from ...file_utils import add_start_docstrings
 from ...utils import logging
-
-# Add modeling imports here
```

Comment:
This should not be removed in this PR as it is used by the model templates to detect where the new model imports should be added.